Lookuptable/testPages/data/export_data_page.py

```diff
@@ -0,0 +1,489 @@
+import os
```

Comment:
Also refer to this class in your testcases instead of the HQSmokeTest one

Reply:
Changes made and pushed the new changes. Thank you !!